test/Prefixer.js

```diff
@@ -3,168 +3,147 @@ import {prefix, compile, serialize} from "../index.js"
+		expect(prefix(`column-span:auto`, 11)).to.equal([`-webkit-column-span:auto;`, `column-span:auto`].join(''))
+		expect(prefix(`column-width:auto`, 12)).to.equal([`-webkit-column-width:auto;`, `column-width:auto`].join(''))
 	})
 
 	test('text', () => {
-		expect(prefix(`text-align:none`)).to.equal([``].join(''))
-		expect(prefix(`text-transform:none`)).to.equal([``].join(''))
-		expect(prefix(`text-shadow:none`)).to.equal([``].join(''))
-		expect(prefix(`text-size-adjust:none`)).to.equal([``].join(''))
-		expect(prefix(`text-decoration:none`)).to.equal([``].join(''))
+		expect(prefix(`text-align:left`, 10)).to.equal([`text-align:left`].join(''))
+		expect(prefix(`text-transform:none`, 14)).to.equal([`text-transform:none`].join(''))
+		expect(prefix(`text-shadow:none`, 11)).to.equal([`text-shadow:none`].join(''))
+		expect(prefix(`text-size-adjust:none`, 16)).to.equal([`-webkit-text-size-adjust:none;`, `-moz-text-size-adjust:none;`, `-ms-text-size-adjust:none;`, `text-size-adjust:none`].join(''))
+		expect(prefix(`text-decoration:none`, 15)).to.equal([`-webkit-text-decoration:none;`, `text-decoration:none`].join(''))
 	})
 
-	test('mask', () => {
-		expect(prefix(`mask-image: none`)).to.equal([``].join(''))
-		expect(prefix(`mask-image: linear-gradient(#fff)`)).to.equal([``].join(''))
+	test.skip('mask', () => {
+		expect(prefix(`mask-image:none`, 10)).to.equal([`-webkit-mask-image:none;`, `mask-image:none;`].join(''))
+		expect(prefix(`mask-image:linear-gradient(#fff)`, 10)).to.equal([`-webkit-mask-image:linear-gradient(#fff);`, `mask-image:linear-gradient(#fff)`].join(''))
 	})
 
 	test('filter', () => {
-		expect(prefix(`filter:grayscale(100%)`)).to.equal([``].join(''))
-		expect(prefix(`fill:red`)).to.equal([``].join(''))
+		expect(prefix(`filter:grayscale(100%)`, 6)).to.equal([`-webkit-filter:grayscale(100%);`, `filter:grayscale(100%)`].join(''))
+		expect(prefix(`fill:red`, 4)).to.equal([`fill:red`].join(''))
 	})
 
 	test('position', () => {
-		expect(prefix(`position: sticky;`)).to.equal([``].join(''))
+		expect(prefix(`position:sticky`, 8)).to.equal([`-webkit-position:sticky;`, `position:sticky`].join(''))
 	})
 
 	test('box', () => {
-		expect(prefix(`box-decoration-break: none;`)).to.equal([``].join(''))
-		expect(prefix(`box-sizing:none;`)).to.equal([``].join(''))
+		expect(prefix(`box-decoration-break:slice`, 20)).to.equal([`-webkit-box-decoration-break:slice;`, `box-decoration-break:slice`].join(''))
+		expect(prefix(`box-sizing:border-box`, 10)).to.equal([`box-sizing:border-box`].join(''))
 	})
 
 	test('clip', () => {
-		expect(prefix(`clip-path: none`)).to.equal([``].join(''))
+		expect(prefix(`clip-path:none`, 9)).to.equal([`-webkit-clip-path:none;`, `clip-path:none`].join(''))
 	})
 
 	test('zoom', () => {
-		expect(prefix(`min-zoom: 0`)).to.equal([``].join(''))
+		expect(prefix(`min-zoom:0`, 8)).to.equal([`min-zoom:0`].join(''))
```

Comment:
Move to line 130-131, i think this is a false positive test for min-inline etc...